src/lib/MuncherSettings.js

```diff
@@ -145,10 +145,10 @@ const MuncherSettings = {
     const daeInstalled = effectModulesAvailable.daeInstalled;
     const advancedMacrosText = isNewerVersion(11, game.version) ? ` Advanced Macros${MuncherSettings.getInstalledIcon("advancedMacrosInstalled")},` : "";
     const itemMacroText = isNewerVersion(11, game.version) ? ` Item Macro${MuncherSettings.getInstalledIcon("itemMacroInstalled")}` : "";
-    const basicEffectsText = `These require DAE${MuncherSettings.getInstalledIcon("daeInstalled")}. For best automation results Midi-QOL${MuncherSettings.getInstalledIcon("midiQolInstalled")},${advancedMacrosText}${itemMacroText} Times Up${MuncherSettings.getInstalledIcon("timesUp")}, and Convenient Effects${MuncherSettings.getInstalledIcon("convenientEffectsInstalled")} are strongly recommended.`;
+    const basicEffectsText = `Requires DAE${MuncherSettings.getInstalledIcon("daeInstalled")}, Midi-QOL${MuncherSettings.getInstalledIcon("midiQolInstalled")}, ${advancedMacrosText}${itemMacroText}, Times Up${MuncherSettings.getInstalledIcon("timesUp")}, and Convenient Effects${MuncherSettings.getInstalledIcon("convenientEffectsInstalled")}.`;
```

Comment:
THis isn't true as the module still gates some effects behind needing DAE for legacy reasons.

Reply:
What is the legacy reason?